Makefile

```
@@ -38,6 +28,7 @@ MACHINE_CONTROLLER_MANAGER_DEPLOYMENT_NAME := machine-controller-manager
 #########################################
 
 include hack/tools.mk
+include .env
```

```suggestion
-include .env
```
this will ignore the .env file if it doesn't exist